<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MacdMarketData</class>
 <widget class="QWidget" name="MacdMarketData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MacdMarketData{background-image: url(:/images/base_back); background-color:red;}
QWidget{background-color:none;}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>801</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QTabWidget::tab-bar  {
        left: 5px; /* move to the right by 5px */
color:white;
    }
     
QTabWidget::pane  {
border:0;
        background:transparent;
    }

    /* Style the tab using the tab sub-control. Note that
        it reads QTabBar _not_ QTabWidget */
    QTabBar::tab  {
        background: rgba(27, 40, 54,0);
        min-width: 8ex;
color:white;
    }
     
    QTabBar::tab:selected, QTabBar::tab:hover  {
        background:  rgba(27, 40, 54,0);
    }
     
    QTabBar::tab:selected  {
        border-top-color: rgba(37, 170, 225, 255);
		border-top-style:solid;
		border-top-width:1px; /* same as pane color */
color:white;
    }
 QTabWidget::tab-bar  {
        left: 5px; /* move to the right by 5px */
color:white;
    }
     
    /* Style the tab using the tab sub-control. Note that
        it reads QTabBar _not_ QTabWidget */
    QTabBar::tab  {
        background: rgba(27, 40, 54,0);
        min-width: 8ex;
color:white;
    }
     
    QTabBar::tab:selected, QTabBar::tab:hover  {
        background:  rgba(27, 40, 54,0);
    }
     
    QTabBar::tab:selected  {
        border-top-color: rgba(37, 170, 225, 255);
		border-top-style:solid;
		border-top-width:1px; /* same as pane color */
color:white;
    }

</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>Summary</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>781</width>
       <height>531</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QFrame" name="topFrame">
        <property name="styleSheet">
         <string notr="true">#topFrame{
border-bottom-color:rgb(37, 170, 225);
border-bottom-style:solid;
border-bottom-width:1px;
border-top-color:rgb(37, 170, 225);
border-top-style:solid;
border-top-width:1px;
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout_3">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>19</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>LAST</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lastValue">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>BID</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="bidValue">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>ASK</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="askValue">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QFormLayout" name="formLayout_4">
           <property name="topMargin">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>24HR HIGH</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="highValue">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>24HR LOW</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>BTC VOLUME</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>MACD VOLUME</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="lowValue">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="btcVolValue">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="macdVolValue">
             <property name="font">
              <font>
               <family>Khmer OS System</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QTableView" name="buyTableView">
          <property name="styleSheet">
           <string notr="true">#buyTableView{
gridline-color: rgba(1, 1, 1, 0);
background-color:rgba(1, 1, 1, 0);
border-color:rgba(1, 1, 1, 0);
border-width:0px;
border-style:solid;}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>190</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>190</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="sellTableView">
          <property name="styleSheet">
           <string notr="true">#sellTableView{
gridline-color: rgba(1, 1, 1, 0);
background-color:rgba(1, 1, 1, 0);
border-color:rgba(1, 1, 1, 0);
border-width:0px;
border-style:solid;}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>190</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>190</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>History</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>801</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QTableView" name="historyTableView">
        <property name="styleSheet">
         <string notr="true">#historyTableView{
gridline-color: rgba(1, 1, 1, 0);
background-color:rgba(1, 1, 1, 0);
border-color:rgba(1, 1, 1, 0);
border-width:0px;
border-style:solid;}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
